mobile/ios/scripts/build_arch.sh

```diff
@@ -38,15 +46,15 @@ export CXXFLAGS="-isysroot ${DEVELOPER}/Platforms/${PLATFORM}.platform/Developer
                 --prefix=/ \
                 $EXTRA_CONFIG
     make -j4 V=0
-    make install V=0 DESTDIR=$REPOROOT/build/${PLATFORM}/${ARCH}
+    make install V=0 DESTDIR=$BUILDDIR/build/${PLATFORM}/${ARCH}
     make distclean
 )
 
 # Build libboost.a stub necessary to create boost.framework
```

Comment:
You can dispose of this piece of code. Boost is headers only and it was a hack to create a library for it. You can also dispose of any piece of code instrumental to create the boost framework.

As said above, we depend on boost to compile measurement-kit. But once it is compiled, boost headers are not needed anymore. Hence we don't need a boost framework.